core/src/test/java/org/apache/brooklyn/core/mgmt/rebind/RebindTestUtils.java

```diff
@@ -238,7 +238,6 @@ public LocalManagementContext buildUnstarted() {
                     classLoader);
             ((RebindManagerImpl) unstarted.getRebindManager()).setPeriodicPersistPeriod(persistPeriod);
             unstarted.getRebindManager().setPersister(newPersister, PersistenceExceptionHandlerImpl.builder().build());
-            unstarted.getHighAvailabilityManager().disabled();
```

Comment:
Not sure about the implications of removing this. It seems reasonable that we execute the HA code-path in our tests, so I think this is fine. I take it that some of our integration tests assumd that it's enabled?